assets/index.css

```diff
@@ -0,0 +1,1107 @@
+  margin-top: 1em;
+}
+
+.edit-list {
+  list-style: none;
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: -1em;
+}
+
+.edit-item {
+  padding: 0.5em 1em;
+  display: inline;
+  font-size: 1.3em;
+}
+
+.edit-item a {
+  color: #818181;
+}
+
+.edit-item a:hover {
+  text-decoration: none;
+  color: #333;
+}
+
+.nav-item a:hover {
+  color: #f72210;
+}
+
+.title-summary a:hover,
+.title-summary-fs a:hover,
+.summary-fs a:hover,
+.info-more-stories11 a:hover {
+  text-decoration: underline;
+  text-decoration-color: #f72210;
+}
+
+.info-debate11 a:hover,
+.info-debate31 a:hover,
+.info-opinion11 a:hover {
+  text-decoration: none;
+  color: rgb(185, 54, 157);
+}
+
+@media (min-width: 768px) {
+  .header-user-icon {
+    display: none;
+  }
+
+  .header-login {
+    display: flex;
+  }
+
+  .header-menu {
+    padding-top: 0;
+  }
+
+  main {
+    display: grid;
+    grid-template-columns: 45% 55%;
+    grid-auto-rows: minmax(auto, auto);
+  }
+
+  .top-story-container {
+    padding-bottom: 0 !important;
+    grid-column: 2;
+    grid-row: 1;
+  }
+
+  .featured-stories-container {
+    grid-column: 1;
+    grid-row: 1/5;
+    margin-top: 0;
+  }
+
+  .turn-container {
+    padding-top: 0 !important;
+    padding-bottom: 0 !important;
+    grid-column: 2;
+    grid-row: 2;
+    margin-top: 0 !important;
+  }
+
+  .culture-travel-container {
+    padding-top: 0 !important;
+    padding-bottom: 0 !important;
+    grid-column: 2;
+    grid-row: 3;
+    margin-top: 0 !important;
+  }
+
+  .more-stories-container {
+    padding-top: 0 !important;
+    overflow-y: scroll;
+    height: 79.5em;
+    margin-top: 0 !important;
+    grid-row: 4;
+    grid-column: 2;
+  }
+
+  .debate-container {
+    justify-content: center;
+    margin-top: 0 !important;
+    grid-row: 5;
+    grid-column: 1 / 3;
+  }
+
+  .opinion-container {
+    margin-top: 0 !important;
+    grid-row: 6;
+    grid-column: 1 / 3;
+  }
+
+  .newsletter-signup-container {
+    grid-column: 1 / 3;
+    grid-row: 7;
+  }
+
+  .magazine {
+    margin-top: 0 !important;
+    grid-column: 1 / 3;
+    grid-row: 8;
+    padding-bottom: 0 !important;
+  }
+
+  .editors-pick {
+    margin-top: 0 !important;
+    grid-column: 1 / 3;
+    grid-row: 9;
+    padding: 0 !important;
+  }
+
+  .picks-container {
+    grid-column: 1 / 3;
+    grid-row: 10;
+  }
+
+  .bottom-page {
+    grid-column: 1 / 3;
+    grid-row: 11;
+  }
+
+  footer {
+    grid-column: 1 / 3;
+    grid-row: 12;
+  }
+
+  .more-stories-container::-webkit-scrollbar {
+    display: none;
+  }
+
+  .main-featured-stories {
+    max-height: 21em;
+  }
+
+  .main-top-story {
+    max-height: 24em;
+  }
+
+  .item-turn1,
+  .cult-item1 {
+    grid-column: 1;
+  }
+
+  .item-turn2,
+  .cult-item2 {
+    grid-column: 2;
+  }
+
+  .item-turn3,
+  .cult-item3 {
+    grid-column: 3;
+  }
+
+  .images-container {
+    display: grid !important;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr;
+  }
```

Comment:
Please make use of bootstrap grid. It is not necessary to use `!important` here.

Reply:
I have to use important because it was allready a bootstrap class that display the element as flex and not as grid